src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/transformation/services/ExceptionRecordToCaseTransformer.java

```diff
@@ -0,0 +1,114 @@
+package uk.gov.hmcts.reform.bulkscanccdeventhandler.transformation.services;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.common.OcrFieldNames;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.model.Address;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.model.CcdCollectionElement;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.model.ScannedDocument;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.transformation.model.in.ExceptionRecord;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.transformation.model.out.CaseCreationDetails;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.transformation.model.out.SuccessfulTransformationResponse;
+
+import java.util.List;
+
+import static java.util.Collections.emptyList;
+import static java.util.stream.Collectors.toList;
+import static uk.gov.hmcts.reform.bulkscanccdeventhandler.common.OcrFieldNames.ADDRESS_LINE_1;
+import static uk.gov.hmcts.reform.bulkscanccdeventhandler.common.OcrFieldNames.ADDRESS_LINE_2;
+import static uk.gov.hmcts.reform.bulkscanccdeventhandler.common.OcrFieldNames.ADDRESS_LINE_3;
+import static uk.gov.hmcts.reform.bulkscanccdeventhandler.common.OcrFieldNames.CONTACT_NUMBER;
+import static uk.gov.hmcts.reform.bulkscanccdeventhandler.common.OcrFieldNames.COUNTRY;
+import static uk.gov.hmcts.reform.bulkscanccdeventhandler.common.OcrFieldNames.COUNTY;
+import static uk.gov.hmcts.reform.bulkscanccdeventhandler.common.OcrFieldNames.DATE_OF_BIRTH;
+import static uk.gov.hmcts.reform.bulkscanccdeventhandler.common.OcrFieldNames.EMAIL;
+import static uk.gov.hmcts.reform.bulkscanccdeventhandler.common.OcrFieldNames.FIRST_NAME;
+import static uk.gov.hmcts.reform.bulkscanccdeventhandler.common.OcrFieldNames.LAST_NAME;
+import static uk.gov.hmcts.reform.bulkscanccdeventhandler.common.OcrFieldNames.LEGACY_ID;
+import static uk.gov.hmcts.reform.bulkscanccdeventhandler.common.OcrFieldNames.POST_CODE;
+import static uk.gov.hmcts.reform.bulkscanccdeventhandler.common.OcrFieldNames.POST_TOWN;
+
+@Service
+public class ExceptionRecordToCaseTransformer {
+
+    public static final String EVENT_ID = "createCase";
```

Comment:
`createCase` event id is used to create Exception record. Better to use a different one here as it may confusion.

Reply:
It's an event in BULKSCAN service case definition.

Reply:
https://github.com/hmcts/bulk-scan-shared-infrastructure/blob/master/docker/ccd-definition-import/data/CCD_Definition_BULK_SCAN.template.xlsx

Reply:
worth including the purpose/context of this event id in the name